plugins/plugin-pullrequest-parent/che-plugin-pullrequest-ide/src/main/java/org/eclipse/che/plugin/pullrequest/client/ContributionMixinProvider.java

```diff
@@ -176,7 +189,12 @@ public void apply(Project project) throws OperationException {
                               public void apply(final PromiseError error)
                                   throws OperationException {
                                 invalidateContext(rootProject);
-                                hidePart();
+                                notificationManager.notify(
+                                    contributeMessages.failedToApplyVSCMixin(
+                                        rootProject.getName(), error.getMessage()),
+                                    FAIL,
+                                    FLOAT_MODE);
+                                contributePart.showStub(contributeMessages.unexpectedError());
```

Comment:
I guess instead of showing error message might n't be helpful at all it is better to log error details into the console and display any simple message like this: _Nothing to show_ 
WDYT?